src/Deprecated/Engine/Logging/DistributedLoggers/ConfigurableForwardingLogger.cs

```diff
@@ -188,67 +219,80 @@ public void Initialize(IEventSource eventSource, int nodeCount)
             Initialize(eventSource);
         }
 
-        private void SetForwardingBasedOnVerbosity()
+        private void SetForwardingBasedOnVerbosity(IEventSource eventSource)
```

Comment:
This is only called once per logger and at initialization time, correct? How are double subscriptions possible?